griptape/tools/pdf_reader/tool.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+import logging
+import json
+from typing import Union
+from attr import define, field
+from griptape.artifacts import BaseArtifact, TextArtifact, ErrorArtifact
+from schema import Schema
+from griptape.core import BaseTool
+from griptape.core.decorators import activity
+
+
+@define
+class PdfReader(BaseTool):
+    include_links: bool = field(default=True, kw_only=True, metadata={"env": "INCLUDE_LINKS"})
+
+    @activity(config={
+        "name": "get_content",
+        "description": "Can be used to get all text content from a PDF",
+        "schema": Schema(
+            str,
+            description="Valid path to a PDF file"
+        )
+    })
+    def get_content(self, value: str) -> BaseArtifact:
+        from PyPDF2 import PdfReader as PDFR
+
+        reader = PDFR(value)
+        text = [p.extract_text() for p in reader.pages]
```

Comment:
Could you wrap those in a try-except and return `ErrorArtifact` if there is an error?

Reply:
Done & squashed.

Reply:
Note: I changed the impl of `PdfRead.get_content(...)`to join the return value to a string because there was actually a warning it was a `List[str]`